public/docs/_examples/server-communication/dart/lib/wiki/wikipedia_service.dart

```diff
@@ -0,0 +1,24 @@
+// #docregion
+import 'dart:async';
+import 'package:angular2/angular2.dart';
+import 'package:jsonpadding/jsonpadding.dart';
```

Comment:
@zoechi would you suggest another name ?
Since JSONP = JSON with Padding and I didnt had a better name for it I decided to call it `jsonpadding`.